gef.py

```diff
@@ -8783,9 +8864,12 @@ def do_invoke(self, args):
 
     @staticmethod
     def heap_base():
-      try:
-        return long(gdb.parse_and_eval("mp_->sbrk_base"))
-      except gdb.error:
+        try:
+            base = long(gdb.parse_and_eval("mp_->sbrk_base"))
+            if base > 0:
```

Comment:
Can this be negative?

Reply:
How so? If `mp_` exists but `sbrk_base` hasn't been set then it will be 0, otherwise it will blow up with a `gdb.error`

Reply:
I'm wondering why the test is `> 0` is stead of `if base != 0`

Reply:
ah yeah it can just be ` != 0`

Reply:
Let's do this, I don't like `> 0` comparisons like this.